src/pipeline.ts

```diff
@@ -220,14 +220,24 @@ export class GitHubWorkflow extends PipelineBase {
 
   private jobForAssetPublish(node: AGraphNode, assets: StackAsset[], options: Context): Job {
     const installSuffix = this.cdkCliVersion ? `@${this.cdkCliVersion}` : '';
-    const relativeToAssembly = (p: string) => path.posix.join(cdkoutDir, path.relative(options.assemblyPath, p));
+    const pathDirectories = this.workflowPath.split(path.sep);
+    const cdkoutDir = path.join(pathDirectories.slice(0, pathDirectories.indexOf('.github')).join(path.sep), 'cdk.out');
```

Comment:
is this too brittle? We already ensure that `this.workflowPath` includes `.github/workflows` earlier, so we are confident that `pathDirectories.indexOf('.github')` succeeds.

But all of this is to ensure that the tests, which have a workflow path of `some/temp/dir/.github/workflows/deploy.yml`, have their asset scripts stored in `some/temp/dir/cdk.out`. Otherwise, these assets get put in the source directory, which is weird.